pkg/stdlib/actions/v1/pkgs/metrics.cue

```diff
@@ -0,0 +1,10 @@
+#PromCheck: {
+	#do:       "promCheck"
+	#provider: "metrics"
+
+	query: string
+  promAddress: string
```

Comment:
Can we add a default promAddress pointed to a service name of prometheus?

Reply:
Will be defined in workflowStepDefinition.